pandas/tests/frame/methods/test_cov_corr.py

```diff
@@ -175,6 +175,15 @@ def test_corr_int(self):
         df3.cov()
         df3.corr()
 
+    @td.skip_if_no_scipy
+    @pytest.mark.parametrize("meth", ["pearson", "spearman"])
```

Comment:
Include "kendall" as well? You may need to widen the DataFrame since it currently has only two columns.